### ISSUE TITLE
[BFW-6887] [BUG] M104.1 should not default to M104 for current tool

### ISSUE THREAD

**Issue by Ro3Deee** [prusa3d/Prusa-Firmware-Buddy#4115]
### Printer model

XL 

### Firmware version

6.0.4

### Describe the bug

M104.1 GCode was introduced to Prusa-Firmware-Buddy by this commit:
https://github.com/prusa3d/Prusa-Firmware-Buddy/commit/652a65f8c176db7069917d13bb51dab6b40769ae
to early set hotend temperature so that the printer can start preheating the tool in advance.

PrusaSlicer started to use M104.1 in place of M104 with this commit: 
https://github.com/prusa3d/PrusaSlicer/commit/c916249f2b0e2d9b50077cf3c2402115426ef4f4

But the current implementation of M104.1 is to ignore the P and Q params ( time in seconds till the temperature S is required)  simply process this command as if it was standard M104. 

Because an M104 command on current tool immediately starts to modify the current temperature, an  M104.1 (that is M104 in fact) on current tool can destroy a print if used with P and Q params in mind.

 So, until m104.1 is properly implemented as requested here #4117 , it shouldn't do anything on current tool.

**Comment by bkerler** [comment]
As the source code already states [here](https://github.com/prusa3d/Prusa-Firmware-Buddy/blob/f5a498ab8d2a42341d0dbeb969b7ae047783e860/src/marlin_stubs/M104_1.cpp#L16-L24), the P and Q parameters are planned to be developed already and meanwhile, if preheating is required, M104 is being run instead. So it isn't implemented right now and thus no bug.

**Comment by bkerler** [comment]
@petrubecheru yes, I agree. Imho as of now, this issue has to be handled within PrusaSlicer. The changes within the printer firmware seem to be pretty huge to match the expected behaviour vs. the current behaviour.

**Comment by CZDanol** [comment]
Is there any example where this actually mattered?

**Comment by CZDanol** [comment]
Got an answer to my previous question here:  https://github.com/prusa3d/Prusa-Firmware-Buddy/issues/4117#issuecomment-2709801423
Internal ticket: BFW-6887
